doc/src/features/Praktikumsplaetze.md

```diff
@@ -11,16 +9,33 @@ Individual [Dienststellengruppen](../glossary.md#dienststelle) can be expanded a
 The individual [Praktikumsstelle](../glossary.md#praktikumsstelle) can also be expanded again. 
 This means that all possible information on a [Praktikumsstelle](../glossary.md#praktikumsstelle) is displayed.
 
-Also on this page there is a button labeled `Hinzufügen`.
+
+## Submission örtliche Ausbilderin
+
+On the page there is a button labeled `Hinzufügen`.
 Clicking this button will open a dialog.
 
 In this dialog, you can choose whether you want to report a [student](./../glossary.md#studentin) (`Studium`) or a [trainee](./../glossary.md#auszubildende) (`Ausbildung`).
 This will open a page with the corresponding form.
 
+## Submission örtliche Ausbildungsleitung
+If the user has the role for [örtliche Ausbildungsleitung](../glossary.md#ortliche-ausbildungsleitung)
+On the page there is a button labeled `Hinzufügen`.
+Clicking this button will open a dialog.
```

Comment:
Deleting the first sentence and pulling "On the page there is a button labeled `Hinzufügen`." back to the top may be a good idea